x-pack/plugin/transform/qa/multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/transform/integration/continuous/TermsGroupByIT.java

```diff
@@ -55,13 +59,15 @@ public TransformConfig createConfig() {
 
         PivotConfig.Builder pivotConfigBuilder = new PivotConfig.Builder();
         pivotConfigBuilder.setGroups(
-            new GroupConfig.Builder().groupBy("event", new TermsGroupSource.Builder().setField("event").setMissingBucket(missing).build())
-                .build()
+            new GroupConfig.Builder().groupBy(
+                "event",
```

Comment:
Shouldn't this be `termsField`?

Reply:
no, this is the output field name, on the next line there is `setField(termsField)`

Having that said, it would be more understandable if input and output use different names